CONTRIBUTING.md

```diff
@@ -33,6 +33,57 @@ Experienced git users can find a suggested workflow and manubot build instructio
 
 Once the answer to the question "did you change the contents of the repository in some way that was meaningful (i.e., not just adding an apostrophe or fixing typos)?" is yes, please add yourself to the author list by following the Authorship Guidelines below.
 
+## Reviewing Someone's Pull Request
+
+Reviewing text that other contributors submit is one of the most important ways you can help.
+Because we are all coming from different fields, there is value in having as many people as possible look at each submission to help new ideas percolate.
+Right now, we are asking for at least two people to look at each new piece of text.
+Some things you can check for when you review someone's pull request:
+- Does it pass all manubot checks? (If you don't know how to check this but want to learn, reach out on gitter!)
+- Is there an issue opened for every paper cited?
+- To the extent that the text overlaps with your own area of expertise, is it correct? 
+Are there any ambiguities, oversimplifications, or topics you think should be clarified?
+- Do you have questions about any of the scientific content inside or outside of your field?
+- Will the text be easy for someone from a different scientific discipline to understand?
+
```

Comment:
Great instructions!

Somewhere in this paragraph, it could be helpful to link to the [instructions for viewing an artifact](https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md#how-can-i-see-my-change) if someone prefers to review the pdf instead of the markdown.

Reply:
I didn't realize that was already up! Edited to add it.